src/constants.ts

```diff
@@ -5,3 +5,7 @@ export const CHAIN_CONTRACT_NAME_BYTES32 = '0x436861696e000000000000000000000000
 export const NUMERIC_MULTIPLIER = 10 ** 18;
 
 export const FIXED_NUMBER_PREFIX = 'FIXED_';
+
+export const INT_PREFIX = 'INT_';
+
+export const MAX_UINT224_HEX = '0x00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
```

Comment:
nl

Reply:
yeah im having issues with this...the code has terminal newlines in my code editor, plus i run the formatters already defined in the package.json script, and what you see is what git is pushing